Changelogs for Digital Twin Manager Microservice 1025.0.0 (#4494)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1025-0-0-tenant-option-for-sharing-asset-and-property-definitions-from-the-enterprise-tenant.md

---

> date: ""

Reviewer: ```suggestion
date: "2026-04-09"
```

title: "New tenant option for sharing asset and property definitions from the Enterprise tenant"
product_area: "Application enablement & solutions"
change_type:
    - value: "change-QHu1GdukP"
      label: "Feature"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2703"
version: "1025.0.0"
---
Managing asset definitions across multiple subtenants previously
required each tenant to maintain its own independent set of definitions.
The new tenant option `definitions.multitenant.sharing.mode` lets you
configure from where asset definitions and MEA definitions are stored and
retrieved across your {{< enterprise-tenant >}} hierarchy.

> For more details, refer to the [documentation](https://cumulocity.com/docs/dtm/).

Reviewer 2: ```suggestion
For more details, refer to the [DTM documentation](https://cumulocity.com/docs/dtm/).
```

> More details can be found in the [documentation]([url](https://c8y-ops-docs.c8y.io/release/develop/dtm/dtm-configuration/#option-definitions.multitenant.sharing.mode)).

Reviewer: ```suggestion
For more details, refer to the [documentation]([url](https://cumulocity.com/docs/dtm/)).
```

---
date: ""

> title: "Tenant option for sharing asset and property definitions from the enterprise tenant"

Reviewer 2: ```suggestion
title: "New tenant option for sharing asset and property definitions from the Enterprise tenant"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-QHu1GdukP"
      label: "Feature"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2703"
version: "1025.0.0"
---

> Managing asset definitions across multiple subtenants previously

Reviewer 2: @lara-c8y These are a lot of details for a change log entry. Is this also described in the docs? Then shortening the text here and adding a link to the docs would be better.

required each tenant to maintain its own independent set of definitions.

> The new tenant option definitions.multitenant.sharing.mode lets you

Reviewer 2: ```suggestion
The new tenant option `definitions.multitenant.sharing.mode` lets you
```

> configure where asset definitions and MEA definitions are stored and

Reviewer 2: ```suggestion
configure from where asset definitions and MEA definitions are stored and
```

> retrieved from across your enterprise tenant hierarchy.

Reviewer 2: ```suggestion
retrieved across your {{< enterprise-tenant >}} hierarchy.
```

The option supports two modes: disabled (default) and enabled. When set
to disabled, definitions are stored directly on each tenant

> independently. When set to enabled on both the enterprise tenant and a

Reviewer 2: ```suggestion
independently. When set to enabled on both the {{< enterprise-tenant >}} and a
```

> subtenant, definitions are stored on the enterprise tenant and made

Reviewer 2: ```suggestion
subtenant, definitions are stored on the {{< enterprise-tenant >}} and made
```

available to all subtenants where the option is enabled. Read requests

> from a subtenant are transparently forwarded to the enterprise tenant.

Reviewer 2: ```suggestion
from a subtenant are transparently forwarded to the {{< enterprise-tenant >}}.
```

> Create, update, and delete requests are only permitted on the enterprise

Reviewer 2: ```suggestion
Create, update, and delete requests are only permitted on the {{< enterprise-tenant >}}
```

> tenant and are rejected on the subtenant. If a tenant does not belong to

Reviewer 2: ```suggestion
and are rejected on the subtenant. If a tenant does not belong to
```

> an enterprise tenant, definitions are stored on the tenant itself and

Reviewer 2: ```suggestion
an {{< enterprise-tenant >}}, definitions are stored on the tenant itself and
```

the sharing rules do not apply.

After updating this tenant option, allow up to 10 minutes for the change

> to take effect. Setting enabled on a subtenant while the enterprise

Reviewer 2: ```suggestion
to take effect. Setting enabled on a subtenant while the {{< enterprise-tenant >}}
```

> tenant is set to disabled is an invalid configuration and results in an

Reviewer 2: ```suggestion
is set to disabled is an invalid configuration and results in an
```
